app/views/business_lookup_LTD.scala.html

```diff
@@ -51,26 +51,12 @@
     }
 }
 
-@hintContent = {
-    <p class="govuk-body">@messages("bc.business-verification.utr.help.answer_1")</p>
-    <p class="govuk-body">@messages("bc.business-verification.utr.help.answer_2")</p>
-}
-
 @helpQuestionAnswer = @{
```

Comment:
This is no longer utilised and should be removed (its reference has been removed on line 70)

Reply:
These message keys have already been removed. I have just pushed another commit tidying up any other redundant message keys.

Reply:
helpQuestionAnswer is the one that needs to be removed since it was only used on line 70 and that has been removed

Reply:
Done now